tests/test_flows.py

```diff
@@ -81,8 +81,21 @@ def tcpdump_start(self):
         if opts.tcpdump:
             outputfile = os.path.join(self.rootdir, 'logs', 'test_{}.pcap'.format(self.running))
             debug("Starting tcpdump, output file {}".format(outputfile))
-            inspect = json.loads(subprocess.check_output(('docker', 'network', 'inspect',
-                                                          'prplMesh-net-{}'.format(self.opts.unique_id))))
+            docker_network = 'prplMesh-net-{}'.format(self.opts.unique_id)
```

Comment:
It will be more appropriate to move this to a separate method.

Reply:
Good point, but it will make rebase difficult so I'll do that later.

Reply:
Actually, I needed to make some other changes anyway that will lead to conflicts during the rebase, so I did it now after all.